pkg/lib/journal.js

```diff
@@ -96,11 +96,17 @@ journal.build_cmd = function build_cmd(/* ... */) {
             options.count = null;
     }
 
-    var cmd = [ "journalctl", "-q", "--output=json" ];
+    var cmd = [ "journalctl", "-q" ];
     if (!options.count)
         cmd.push("--no-tail");
     else
         cmd.push("--lines=" + options.count);
+
+    if (!options.output)
+        cmd.push("--output=json");
+    else
+        cmd.push("--output=" + options.output);
```

Comment:
Style hint for default values: `cmd.push("--output=" + (options.output || "json"))`